extensions/shared/external-media/sources/with-media.js

```diff
@@ -68,7 +71,10 @@ export default function withMedia() {
 				 * This can be removed once
 				 * https://github.com/WordPress/gutenberg/issues/22940 is fixed.
 				 */
-				if ( [ UP, DOWN, LEFT, RIGHT ].includes( event.keyCode ) ) {
+				if (
+					[ UP, DOWN, LEFT, RIGHT ].includes( event.keyCode ) &&
+					! event.target.classList.contains( 'jetpack-external-media-browser__media__item' ) // Only let arrow key navigation on media grid items through. All others need to be stopped.
```

Comment:
I don't like that this uses a class check. The gist, [as explained earlier](https://github.com/Automattic/jetpack/pull/16118#discussion_r439028722) is that we can _only let arrow key presses on media grid items avoid this `stopPropogation` call_. 

The question is, **how do we guarantee that the `event.target` really is a media item**? 

- Is there a way to access its ref?
- Should we use the HTML instead, like a `li` or `role="checkbox"` check? This seems more likely to get missed though.